kDriveCore/Data/UploadQueue/Servicies/PhotoLibraryUploader+Sync.swift

```diff
@@ -20,18 +20,24 @@ import Foundation
 import RealmSwift
 
 public extension PhotoLibraryUploader {
```

Comment:
TODO: PhotoLibraryUploader should have its own Transactionable instance to work with the DB instead of doing magic with a realm from a background realm or something

Reply:
done